Description: fix spelling errors in kcmsystemd.cpp
Author: Shawn Sörbom
Last-Update: 2014-09-28
--- a/src/kcmsystemd.cpp
+++ b/src/kcmsystemd.cpp
@@ -616,7 +616,7 @@
 
 void kcmsystemd::defaults()
 {
-  if (KMessageBox::warningYesNo(this, i18n("Load deafult settings for all tabs?")) == KMessageBox::Yes)
+  if (KMessageBox::warningYesNo(this, i18n("Load default settings for all tabs?")) == KMessageBox::Yes)
   { 
     //defaults for system.conf
     for (int i = 0; i < confOptList.size(); ++i)
@@ -699,7 +699,7 @@
     else // Other error
       KMessageBox::error(this, i18n("Unable to write the (%1) file:\n%2", reply.data()["filename"].toString(), reply.data()["errorDescription"].toString()));
   } else // Writing succeeded
-    KMessageBox::information(this, i18n("Configuration files succesfully written to: %1", helperArgs["etcDir"].toString()));
+    KMessageBox::information(this, i18n("Configuration files successfully written to: %1", helperArgs["etcDir"].toString()));
 }
 
 void kcmsystemd::slotUpdateConfOption()
